.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+.env
```

Comment:
There's a specific syntax to allow you to include all files in a single line and exclude `.env.example`